prow/manifests/overlays/metal3/config.yaml

```diff
@@ -1394,39 +1234,6 @@ presubmits:
               - make
             image: docker.io/golang:1.20
             imagePullPolicy: Always
-    - name: gofmt
-      branches:
-        - main
-      skip_if_only_changed: '(((^|/)OWNERS)|(\.md))$'
-      decorate: true
-      spec:
-        containers:
-          - args:
-              - ./hack/gofmt.sh
-            command:
-              - sh
-            env:
-              - name: IS_CONTAINER
-                value: "TRUE"
-            image: docker.io/golang:1.21
-            imagePullPolicy: Always
-    - name: gofmt
-      branches:
-        - release-1.5
-        - release-1.6
-      skip_if_only_changed: '(((^|/)OWNERS)|(\.md))$'
-      decorate: true
-      spec:
-        containers:
-          - args:
-              - ./hack/gofmt.sh
-            command:
-              - sh
-            env:
-              - name: IS_CONTAINER
-                value: "TRUE"
-            image: docker.io/golang:1.20
-            imagePullPolicy: Always
     - name: gofmt
       branches:
         - release-1.4
```

Comment:
In first glance this also seems the same case as CAPM3, any reason to leave it here?

Reply:
also gosec for release-1.4 inline 1354

Reply:
Checking...
We have [golangci-lint prow job](https://github.com/metal3-io/project-infra/blob/7b988cb551aef97114c0864b33bdd8c166b0ec06/prow/manifests/overlays/metal3/config.yaml#L1470). This calls [ensure-golanci-lint.sh](https://github.com/metal3-io/ip-address-manager/blob/release-1.4/hack/ensure-golangci-lint.sh), which calls [make lint](https://github.com/metal3-io/ip-address-manager/blob/5f16ccffb25a384d873e556129c559ded28687c7/Makefile#L136-L139) that uses golangci-lint. It has [gofmt, gosec and govet enabled](https://github.com/metal3-io/ip-address-manager/blob/5f16ccffb25a384d873e556129c559ded28687c7/.golangci.yaml#L22-L27). Looks like we can remove it :heavy_check_mark: